gprofiler/profilers/python_ebpf.py

```diff
@@ -213,6 +215,36 @@ def start(self) -> None:
             raise
         else:
             self.process = process
+            self._register_process_selectors()
+
+    def _register_process_selectors(self) -> None:
+        self.process_selector = selectors.DefaultSelector()
+        assert self.process_selector and self.process and self.process.stdout and self.process.stderr  # for mypy
+        self.process_selector.register(self.process.stdout, selectors.EVENT_READ)
+        self.process_selector.register(self.process.stderr, selectors.EVENT_READ)
+
+    def _unregister_process_selectors(self) -> None:
+        assert self.process_selector
+        for selector_key in list(self.process_selector.get_map().values()):
+            self.process_selector.unregister(selector_key.fileobj)
```

Comment:
Is there a reason to keep the `self.process_selector` alive and not set it to `None`?